frontend/src/scenes/sessions/SessionsView.tsx

```diff
@@ -104,6 +104,12 @@ export function SessionsView({ personIds, isPersonPage = false }: SessionsTableP
         {
             title: 'Session duration',
             render: function RenderDuration(session: SessionType) {
+                if (session.session_recordings.length > 0) {
+                    const sum = session.session_recordings
+                        .map(({ recording_duration }) => recording_duration)
```

Comment:
Not sure if we need to do `recording_duration || 0` in case it's `null` or `undefined` from the backend?

Reply:
It should always have some value, but to be sure, I added a `or 0` in the backend.